proptest-macro/src/property_test/tests/snapshots/proptest_macro__property_test__tests__snapshot_tests__basic_derive_example.snap

```diff
@@ -0,0 +1,1291 @@
+                    },
+                    Punct {
+                        char: ',',
+                        spacing: Alone,
+                    },
+                ],
+            },
+            Punct {
+                char: ';',
+                spacing: Alone,
+            },
+            Ident {
+                sym: match,
+            },
+            Ident {
+                sym: result,
+            },
+            Group {
+                delimiter: Brace,
+                stream: TokenStream [
+                    Ident {
+                        sym: Ok,
+                    },
+                    Group {
+                        delimiter: Parenthesis,
+                        stream: TokenStream [
+                            Group {
+                                delimiter: Parenthesis,
+                                stream: TokenStream [],
+                            },
+                        ],
+                    },
+                    Punct {
+                        char: '=',
+                        spacing: Joint,
+                    },
+                    Punct {
+                        char: '>',
+                        spacing: Alone,
+                    },
+                    Group {
+                        delimiter: Brace,
+                        stream: TokenStream [],
+                    },
+                    Ident {
+                        sym: Err,
+                    },
+                    Group {
+                        delimiter: Parenthesis,
+                        stream: TokenStream [
+                            Ident {
+                                sym: e,
+                            },
+                        ],
+                    },
+                    Punct {
+                        char: '=',
+                        spacing: Joint,
+                    },
+                    Punct {
+                        char: '>',
+                        spacing: Alone,
+                    },
+                    Ident {
+                        sym: panic,
+                    },
+                    Punct {
+                        char: '!',
+                        spacing: Alone,
+                    },
+                    Group {
+                        delimiter: Parenthesis,
+                        stream: TokenStream [
+                            Literal {
+                                lit: "{}",
+                            },
+                            Punct {
+                                char: ',',
+                                spacing: Alone,
+                            },
+                            Ident {
+                                sym: e,
+                            },
+                        ],
+                    },
+                    Punct {
+                        char: ',',
+                        spacing: Alone,
+                    },
+                ],
+            },
+        ],
+    },
+]
```

Comment:
is there an equivalent of a snapshot test of the printed output? this tree i find tough to interpret directly -- though it will for sure be helpful as things change, to have a basis to compare to

Reply:
Decided to run it through a formatter first - much more readable now